vhdbuilder/packer/import-image.sh

```diff
@@ -15,13 +18,24 @@ do
     fi
 done
 
+if [[ $HYPERV_GENERATION == "V2" ]]; then
+    IMPORT_IMAGE_URL=${IMPORT_IMAGE_URL//-gen1-/-gen2-}
+fi
+if [[ $HYPERV_GENERATION == "V1" ]]; then
+    IMPORT_IMAGE_URL=${IMPORT_IMAGE_URL//-gen2-/-gen1-}
+fi
```

Comment:
we should let the pipeline to pass in the right url for v1, v2 respectively. Relying on the implicit naming pattern is not necessary and will be confusing. Simple is the best.